src/main/webapp/css/vacation-overview.css

```diff
@@ -108,10 +108,86 @@
   font-size: 12px;
 }
 
+.vacationOverview-table th,
 .vacationOverview-table td {
-  border: 1px solid #FFF;
+  color: #444444 !important;
+  border: 1px solid #ffffff;
+  font-weight: normal;
 }
 
-.vacationOverview-tbody tbody {
-  border: 1px solid #000;
+.vacationOverview-table td {
+  text-align: center;
+}
+
+.vacationOverview-table thead th {
+  padding: 8px;
+  cursor: default !important; /* override $.tablesorter */
+}
+
+.vacationOverview-table tbody th {
+  padding-top: 8px;
+  padding-bottom: 8px;
+  padding-left: 0;
+  padding-right: 16px;
+}
+
+@media print {
+
+  .vacationOverview-table th,
+  .vacationOverview-table td {
+    /*border: none !important;*/
+    border: 1px solid #e2e8f0 !important;
+  }
+
+  .vacationOverview-table thead th {
+    font-weight: 600 !important;
+    padding: 4pt 8pt;
+  }
+  .vacationOverview-table tbody th {
+    font-weight: 300 !important;
+    padding: 4pt 8pt;
+  }
+
+  .vacationOverview-table-legend th {
+    text-align: center;
+    padding: 0 12pt;
+    font-weight: 300 !important;
+  }
+
+  .vacationOverview-day-personal-holiday-status-WAITING {
+    color: #b7791f !important;
+    background-color: #fffff0 !important;
+  }
+  .vacationOverview-day-personal-holiday-half-day-status-WAITING-morning {
+    color: #b7791f !important;
```

Comment:
There is no difference between full/morning/noon, maybe `linear-gradient` is missing?

Reply:
additionally, using the chrome browser on mac I have to enable 'background graphics' in the print dialog. otherwise no colors are printed. imho we can't rely on the user knowing this.

![Screenshot 2020-08-17 at 19 10 41](https://user-images.githubusercontent.com/1732200/90423805-6966b180-e0bd-11ea-8c03-3ad7e5d83e90.png)